UnitTests/CommonTestUtils/ReferenceRepository.cs

```diff
@@ -23,6 +23,8 @@ public ReferenceRepository()
 
         public string CommitHash => _commitHash;
 
+        public string CreateRepoFile(string fileName, string fileContent) => _moduleTestHelper.CreateRepoFile(fileName, fileContent);
```

Comment:
Please move before `CreateTag`

Reply:
Makes sense, thanks! May I ask you if you use some tool to check this or just common sense? 
I've no ReSharper unfortunately but I've tried a free tool _CodeMaid_ to reorganize the file but it has found much more codeblocks to move/reorganize.

Reply:
No tool unfortunately. I strive to follow the old [.NET Framework guidelines](https://docs.microsoft.com/en-us/archive/blogs/brada/internal-coding-guidelines#28-file-organization) but unless it is enforced by tools it is an uphill battle.

Reply:
I see, thanks anyway for the link! Actually, the above mentioned VS extension seems to do a good job in reordering the code (and it's configurable) but I miss the option to sort the new code blocks only (kind of git-diff based). Hmm, could be a nice extension for VS or even GE! :thinking: